### PR TITLE
🦆 make nexus setup more robust 🦆

### PR DESCRIPTION
**Pull request by eformat**
#### What is this PR About?
make the nexus-setup job more robust. add service api check as well as the existing pod check.
we had a failure on install recently where pod was up, but api was not ready. 

#### How do we test this?
helm install chart

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

**Review by pabrahamsson — APPROVED** [review]
LGTM
Waiting on @ckavili to review before merging (since she was requested)

**Comment by ckavili** [comment]
LGTM as well 🦋

**Pull request merged**
